test/integration/rtds_integration_test.cc

```diff
@@ -177,9 +177,9 @@ TEST_P(RtdsIntegrationTest, RtdsReload) {
   EXPECT_EQ("saz", getRuntimeKey("baz"));
 
   EXPECT_EQ(0, test_server_->counter("runtime.load_error")->value());
-  EXPECT_EQ(0, test_server_->counter("runtime.update_failure")->value());
+  EXPECT_EQ(0, test_server_->counter("runtime.some_rtds_layer.update_failure")->value());
   EXPECT_EQ(initial_load_success_ + 2, test_server_->counter("runtime.load_success")->value());
-  EXPECT_EQ(2, test_server_->counter("runtime.update_success")->value());
+  EXPECT_EQ(2, test_server_->counter("runtime.some_rtds_layer.update_success")->value());
```

Comment:
These may be breaking changes for various people's monitoring setups. I don't know if this should be flag-flipped so people can have a grace period when this goes live before updating their monitoring.

Reply:
Yeah this is a breaking change potentially when it comes to these stat names. I guess we could 1) config guard this change 2) runtime guard it and hope for the best? 3) add a stat_prefix to each layer so that the component is directly configurable for each yaml

Having different layers overwrite each others stats doesn't seem very desirable so I'm leaning towards 2, thoughts?

Reply:
I was thinking of a runtime guard.

/wait